src/screens/About/About.tsx

```diff
@@ -239,9 +239,9 @@ const About = ({ children }: { children: React.ReactNode }) => {
             1.0. This means that, on average, each person with COVID will
             transmit to 0.9 other people, which means the disease is shrinking.
             An R<sub>0</sub> of 1.0 means COVID is in perfect equilibrium,
-            neither growing nor shrinking. And an R<sub>0</sub> greater than 1.0
-            means that COVID is growing exponentially — and the greater the
-            value above 1.0, the faster the doubling speed.
+            neither growing nor shrinking. An R<sub>0</sub> greater than 1.0
+            means that COVID is growing exponentially. The greater the value
+            above 1.0, the faster the doubling speed.
```

Comment:
Sorry to flag text you didn't change, but it feels like this whole paragraph should be saying R-effective instead of R<sub>0</sub>?

My armchair epidemiology understanding is that R<sub>0</sub> is the "initial conditions" value of R.  You can't "drive it down".

But I'm sure the public doesn't understand, so maybe we don't care.